app/components/common/menu_id/classification/Classification.js

```diff
@@ -1,113 +1,123 @@
 import CLASSIFICATION_LABELS from "@/components/enums/classification_labels";
+import { useSelector } from "react-redux";
 
 export default function Classification(props) {
+  let current = null;
+  if (props.page === "trip") {
+    current = useSelector((state) => state.trip.selectedTrip);
+  } else if (props.page === "tour") {
+    current = useSelector((state) => state.tour.selectedTour);
+  }
+
   return (
     <div className="p-8 m-auto">
       <div>
-        {props.percentageAcessibility ? (
+        {current.percentageAcessibility ? (
```

Comment:
what if current is null?

`current?.percentageAccessibility ?`

Reply:
please change this everywhere else

Reply:
well noted! Thank you
Changes done